src/WinRT.Runtime/Projections/IPropertyValue.net5.cs

```diff
@@ -325,12 +326,46 @@ private static T CoerceValue<T>(object value)
                 {
                     return (T)(object)guid.ToString("D", global::System.Globalization.CultureInfo.InvariantCulture);
                 }
+                else if (typeof(T) == typeof(byte))
```

Comment:
Just so I'm sure I follow, how come we didn't need the 9x9 combinations anymore and just 9 instead?

Reply:
We're not assuming anything about the passed in `value` type since we need to support user-defined-IConvertible (and other BCL-provided `IConvertible` implementations) anyway. We're just using the `Convert.ToX` methods instead of generic math since we'd need to use the `Convert.ToX` methods anyway and practically that branch would never be trimmable, no matter how smart we make ILC and RyuJIT.

Reply:
I guess I'm still not sure I get it. How is this change allowing all that extra stuff to be trimmed out then? 🤔

Reply:
We never call `Convert.ToDateTime`, and a number of the other `ToX` methods, so we can trim out the interface definition and all of the implementations of all of those methods. `IConvertible.ToDateTime` in particular is the expensive one.

Reply:
Ah, gotcha, makes sense. Thank you! 🙂